README.md

```diff
@@ -23,31 +27,121 @@ const job = await client.createReport({
   endTime: new Date().toISOString(),
   browserName: "Chrome",
   browserVersion: "105",
-  framework: "playwright",
+  framework: "playwright", // alternatives: cypress, playwright, testcafe
   frameworkVersion: "1.25.0",
   platformName: "Windows 11"
 });
 console.log(job.id); // the job ID
 console.log(job.url); // the full URL of the job
 ```
 
-## Upload Assets
+## Upload log for JavaScript Batch Frameworks
+
+The minimum `sauce-test-report.json` file to fullfill the web UI:
+
+```json
+{
+  "status": "passed",
+  "attachments": [],
+  "suites": [
+    {
+      "name": "somegroup",
+      "status": "passed",
+      "metadata": {},
+      "suites": [
+        {
+          "name": "somefile.test.js",
+          "status": "passed",
+          "metadata": {},
+          "suites": [],
+          "attachments": [],
+          "tests": [
+            {
+              "name": "yay",
+              "status": "passed",
+              "duration": 123,
+              "startTime": "2023-07-06T22:03:19.591Z",
+              "attachments": [],
+              "metadata": {}
+            }
+          ]
+        }
+      ],
+      "attachments": [],
+      "tests": []
+    }
+  ],
+  "metadata": {}
+}
+```
+To upload `sauce-test-report.json`:
 
 ```javascript
-const Readable = require('stream').Readable;
+const uploads = await client.uploadAssets(
+  job.id,
+  [{
+    filename: "log.json",
+    data: fs.createReadStream("log.json")
```

Comment:
Aren't you [trying to upload](https://github.com/saucelabs/node-testcomposer/pull/12/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R77) "sauce-test-report.json"?

Reply:
ah.. Nice catch!